.github/actions/python-environment/action.yml

```
@@ -18,22 +18,44 @@ inputs:
     required: true
```

wrote more in thread, but this was an excellent point you had.
i'd leave the others two `required: true` to communicate the intent of providing them
```suggestion
    required: false
```